addons/point_of_sale/static/src/xml/Screens/PaymentScreen/PaymentScreen.xml

```diff
@@ -87,7 +87,7 @@
                                     t-on-click="toggleIsToInvoice">
                                     <i class="fa fa-file-text-o" /> Invoice
                                 </div>
-                                <div t-if="env.pos.config.tip_product_id" class="button js_tip"
+                                <div t-if="env.pos.config.iface_tipproduct and env.pos.config.tip_product_id" class="button js_tip"
```

Comment:
You need to rebase. During rebase, you'll have to replace the t-if with `pos.globalState.config.iface_tipproduct and pos.globalState.config.tip_product_id`.